### ISSUE TITLE
Seperate keybindings tied to filters

### ISSUE THREAD

**Issue by oysandvik94** [laktak/extrakto#123]
Usually I know whether I want to copy a word or a line before I trigger Extracto. It would be great if I could have one keybinding for each filter, instead of having to change filter once the fzf window is open.

I wouldn't mind trying to make a PR if this is a feature that makes sense.

**Comment by laktak** [comment]
@marcdeop implemented a feature in #118 to set the initial filter with a second argument to open.sh.

I haven't gotten around to document this in the README yet. It works like this:

```
tmux bind-key YOUR-KEY run-shell "~/.tmux/plugins/extrakto/scripts/open.sh \"#{pane_id}\" FILTER-NAME"
```

Would that work for you?

**Comment by oysandvik94** [comment]
Sorry for the late answer, that's exactly what I needed. Thanks!
